src/migrateCalculatedMeasuresToActivePivot/removeCalculatedMemberDefinitionFromMDXAndGetCubeName.ts

```diff
@@ -0,0 +1,48 @@
+import { DataModel, CubeName, getCube } from "@activeviam/data-model";
+import {
+  MdxString,
+  MdxSelect,
+  getCalculatedMeasures,
+  getCubeName,
+  removeCalculatedMemberDefinition,
+  stringify,
+} from "@activeviam/mdx";
+import { parse } from "@activeviam/activeui-sdk";
+import _intersection from "lodash/intersection";
+
+export const removeCalculatedMemberDefinitionFromMDXAndGetCubeName = (
+  mdx: MdxString,
+  namesOfCalculatedMeasurestoMigrate: string[],
+  dataModel: DataModel,
+): {
+  calculatedMeasuresWithCubeNames: { [measureName: string]: CubeName };
```

Comment:
This makes sense to have this structure accross several MDX queries (= several widgets), but here for a single MDX string there is only one cube name.
So the return type could be instead:
`{cubeName: CubeName, namesOfMigratedCalculatedMeasures: string[], migratedMdx: MdxString}`
And the variable names and logic can be adapted accordingly.